OracleWebLogic/samples/12213-webtier-apache/Dockerfile

```diff
@@ -1,61 +1,92 @@
 # Example of Apache HTTP Server with WebLogic plugin for load balancing WebLogic on Docker Containers
 #
-# Copyright (c) 2016-2019 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2016-2020 Oracle and/or its affiliates.
```

Comment:
Copyright (c) 2016-2020 Oracle and/or its affiliates. -> Copyright (c) 2016, 2020, Oracle and/or its affiliates.